libraries/base/GHC/IO/Encoding/Iconv.hs

```diff
@@ -58,12 +59,12 @@ iconv_trace s
 
 {-# NOINLINE localeEncodingName #-}
```

Comment:
And we do use that, `c_localeEncoding` was implemented sometime yesterday :cat2: